public/index.html

```diff
@@ -15,6 +15,15 @@
   <link rel="manifest" href="%PUBLIC_URL%/favicon/manifest.json">
   <link rel="mask-icon" href="%PUBLIC_URL%/favicon/safari-pinned-tab.svg" color="#1f2666">
   <meta name="theme-color" content="#ffffff">
+
+  <!-- Google Tag Manager -->
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
Can we configure the app so that during development we don't send data to FullStory (either by configuring or by removing the Google Tag Manager tag)?

Reply:
Just wrap it into a component.
First two links from google:
https://github.com/holidaycheck/react-google-tag-manager
https://github.com/alinemorelli/react-gtm

Reply:
I didn't use those components because Google says it's important for the <script> tag to be as high in the <head> tag as possible and React gets loaded in the <body>. Fortunately the template can read environment variables so I'll add this:
```
<% if ('%NODE_ENV%' === 'production') { %>
(include the tag)
<% } %>
```

Reply:
Just pushed that change. Had to do this instead of the above:
```<% if (process.env.NODE_ENV === 'production') { %>```